tests/test_apdbCassandra.py

```diff
@@ -103,10 +103,6 @@ def tearDown(self) -> None:
         query = f"DROP KEYSPACE {self.keyspace}"
         self._run_query(query)
 
-    if TYPE_CHECKING:
-        # For mypy.
-        def make_instance(self, **kwargs: Any) -> ApdbConfig: ...
-
```

Comment:
Not sure why this was removed. Should it be a separate commit?

Reply:
It was a leftover for older refactoring, should have removed it then. I can split the commit.